RealEstate/apps/core/templates/core/houseEval.html

```diff
@@ -0,0 +1,105 @@
+{% extends "core/base.html" %}
+{% load bootstrap3 %}
+{% load staticfiles %}
+
+{% block style %}
+<link rel="stylesheet" type="text/css" href="{% static "slide.css" %}">
+<style>
+.slider-selection {
+	background: #BABABA;
+}
+.vcenter {
+	display: inline-block;
+	vertical-align: middle;
+	float: none;
+	margin-left:-3px;
+}
+.alert-success {
+	margin-top: 30px;
+}
+ul {
+	margin-top:50px;
+}
+.inner-rating {
+	padding:20px 0;
+}
+</style>
+{% endblock %}
+
+{% block body %}
+	<h2><strong> Evaluation for {{house}}</strong> </h2>
+	<h4><em>{{house.address}}</em></h4>
+	<ul class="list-group">
+		{% for category, score in grades %}
+			<li class="list-group-item">
+				<div class="row">
+					<div class="col-sm-6 vcenter">
+						<h4>{{category.summary}}</h4>
+						<p>
+							{% if category.description %}
+								{{ category.description }}
```

Comment:
I would reverse the order of the <p> tag and the 'if category' check, otherwise there is sometimes an empty <p> tag for no reason.